Sources/StreamChatUI/UIConfig.swift

```diff
@@ -16,6 +16,8 @@ public struct _UIConfig<ExtraData: ExtraDataTypes> {
 
     /// An avatar view with an online indicator.
     public var presenceAvatarView: _ChatPresenceAvatarView<ExtraData>.Type = _ChatPresenceAvatarView<ExtraData>.self
+    
+    public var titleView: _ChatMessageListTitleView<ExtraData>.Type = _ChatMessageListTitleView<ExtraData>.self
```

Comment:
This should ideally do inside the `MessageListUI` section

Reply:
👍